GeosCore/hco_interface_gc_mod.F90

```diff
@@ -4891,12 +4903,11 @@ SUBROUTINE Compute_Sflx_for_Vdiff( Input_Opt,  State_Chm, State_Diag,      &
        !=====================================================================
        ! Defining Satellite Diagnostics
        !=====================================================================
-
        ! Define emission satellite diagnostics
        IF ( State_Diag%Archive_SatDiagnColEmis ) THEN
           DO S = 1, State_Diag%Map_SatDiagnColEmis%nSlots
              N = State_Diag%Map_SatDiagnColEmis%slot2id(S)
-             State_Diag%SatDiagnColEmis(:,:,S) = colEflx(:,:,N)
+             State_Diag%SatDiagnColEmis(I,J,S) = colEflx(I,J,N)
```

Comment:
This is a bug fix we should note in the changelog. These diagnostics must have been many times larger than they should be before this fix.

Reply:
@lizziel: We can test this to make sure we get zero-diff output for this diagnostic